src/System/Wlog/LoggerConfig.hs

```diff
@@ -256,12 +268,14 @@ topLevelParams =
 
 instance FromJSON LoggerConfig where
     parseJSON = withObject "rotation params" $ \o -> do
-        _lcRotation      <-         o .:? "rotation"
-        _lcTermSeverity  <-         o .:? "termSeverity"
-        _lcShowTime      <- Any <$> o .:? "showTime"    .!= False
-        _lcShowTid       <- Any <$> o .:? "showTid"     .!= False
-        _lcFilePrefix    <-         o .:? "filePrefix"
-        _lcTree          <- parseJSON $ Object $ filterObject topLevelParams o
+        _lcRotation        <-                         o .:? "rotation"
+        -- TODO: Support word "All" which will include all severities at once
+        _lcTermSeverityOut <- (fmap Set.fromList) <$> o .:? "termSeveritiesOut"
+        _lcTermSeverityErr <- (fmap Set.fromList) <$> o .:? "termSeveritiesErr"
```

Comment:
@vrom911 Wouldn't you mind to fix issue #32 as well in this PR? Please, ask questions if you need more details on this!

Reply:
Okay, I'll have a look at this one as well, thanks! :wink: